inlong-sdk/transform-sdk/src/test/java/org/apache/inlong/sdk/transform/process/TestTransformArithmeticFunctionsProcessor.java

```diff
@@ -55,6 +55,37 @@ public class TestTransformArithmeticFunctionsProcessor {
         kvSink = new KvSinkInfo("UTF-8", dstFields);
     }
 
+    @Test
+    public void testModuloFunction() throws Exception {
+        String transformFunctionSql = "select mod(numeric1,100) from source";
+        String transformExpressionSql = "select numeric1 % 100 from source";
+        TransformConfig functionConfig = new TransformConfig(transformFunctionSql);
+        TransformProcessor<String, String> functionProcessor = TransformProcessor
+                .create(functionConfig, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        TransformConfig expressionConfig = new TransformConfig(transformExpressionSql);
+        TransformProcessor<String, String> expressionProcessor = TransformProcessor
+                .create(expressionConfig, SourceDecoderFactory.createCsvDecoder(csvSource),
+                        SinkEncoderFactory.createKvEncoder(kvSink));
+        // case1: "3.1415926|4a|4|8"
```

Comment:
It is better to indicate the corresponding function expression rather than the data in the case comment
```suggestion
        // case1: mod(3.1415926,100)
```

Reply:
done